### PR TITLE
V0.1.0

### PR DESCRIPTION
**Pull request by aneliya-n-ivanova**

### CODE REVIEW

File: helm_chart/ansible-operator/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: ansible-operator
+description: ansible-operator installation chart
```
Comment (mimiteto) [inline helm_chart/ansible-operator/Chart.yaml:3]: Enrich this. There are several more options that may make your chart more user friendly [ref](https://helm.sh/docs/topics/charts/).

File: operator/executor_start.sh [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    done
+
+    logs_format info "SSH client configured, setting restricted permissions for file /home/${SSH_USER_NAME}/.ssh/config"
+    chmod 0600 "/home/${SSH_USER_NAME}/.ssh/config"
+}
+
+function git_repo_setup(){
+    logs_format info "Attempting to clone git repository ${GIT_REPO} with ${RETRIES} retries"
+    for retry in $(seq 1 "${RETRIES}"); do
+        if git clone "${GIT_REPO}" "${PLAYBOOK_CONFIG_PATH}/repo" &>/dev/null;
```
Comment (mimiteto) [inline operator/executor_start.sh]: Make it so we are cloning only a specific commit (e.g. specified tag or latest commit in a branch).

File: operator/roles/ansible-executor/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#TODO: Add role documentation
```
Comment (mimiteto) [inline operator/roles/ansible-executor/README.md]: #FixMe

File: operator/roles/ansible-executor/meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
```
Comment (mimiteto) [inline operator/roles/ansible-executor/meta/main.yml:1]: #FixMe

File: operator/roles/ansible-executor/README.md (hunk 3 above)
Comment (aneliya-n-ivanova) [reply operator/roles/ansible-executor/README.md]: docs/ANSIBLE_EXECUTOR.md

**Pull request merged**